getting-started/git-boot-camp.rst

```diff
@@ -44,11 +44,14 @@ You will only need to do this once.
 
 1. Go to https://github.com/python/cpython.
 
-2. Press ``Fork`` on the top right.
+2. Press ``Fork`` located near the top right of the page.
 
-3. When asked where to fork the repository, choose to fork it to your username.
+   * The required fields ``Owner`` and ``Repository name`` should already be pre-populated with your username and the original repository name "cpython".
+   * Uncheck the "Copy the ``main`` branch only" button.
```

Comment:
And we could make this its own step:
```suggestion
3. Uncheck the "Copy the ``main`` branch only" button.
```

Reply:
Should we recommend this at all?
Other branches can be always fetched later, if/when needed.
Unchecking this will also make the initial cloning slower, and most contributors won't even need other branches.

Reply:
If this isn't unchecked, then subsequent instructions in the guide will be in error or incorrect.